plugins/inventory/nb_inventory.py

```diff
@@ -1505,7 +1505,8 @@ def fetch_api_docs(self):
         except Exception:
             openapi = {}
 
-        cached_api_version = openapi.get("info", {}).get("version")
+        if cached_api_version := openapi.get("info", {}).get("version"):
+            cached_api_version = ".".join(cached_api_version.split(".")[:2])
```

Comment:
I would prefer to avoid using the `:=` operator. This would be the first usage in our entire repo, so would you be willing to rewrite it, to not use `:=`? Thanks

Reply:
You bet, done.